### DISCUSSION TITLE
2b2)

### DISCUSSION THREAD

**Discussion by shadowbourne** [wenceorg/comp4187#99] [Q&A] [answered]
Three quick queries:
* firstly, are we applying neumann on top of dirichlet or seperately?
* What does n mean ? A.k.a what does the notation $\nabla u \cdot n$ refer to?

Does it mean at the point n? If so is n the end boundary? or is it any individual point n? Or is it both boundaries?

* is h the step size? or is it anything?

Thanks!

**Comment by annereinarz** [comment, marked as answer]
1) Separately, you can't have both in one place.
2) That is the outer normal, so $\nabla u \cdot n$ is the derivative into the direction of that boundary.
3) h is a function that gives the Neumann condition (same as g for Dirichlet)
